localstack-core/localstack/services/lambda_/event_source_mapping/senders/lambda_sender.py

```diff
@@ -59,7 +59,16 @@ def send_events(self, events: list[dict] | dict) -> dict:
             InvocationType=invocation_type,
             **optional_qualifier,
         )
-        payload = json.load(invoke_result["Payload"])
+
+        try:
+            payload = json.load(invoke_result["Payload"])
+        except json.JSONDecodeError:
+            payload = None
```

Comment:
Not sure if we should be setting this to `None` -- what if the payload is valid but just not JSON?